web/templates/recent_view_filters.hbs

```diff
@@ -1,4 +1,3 @@
-<button data-filter="all" type="button" class="btn btn-default btn-recent-filters">{{t 'All' }}</button>
```

Comment:
You need to adjust corresponding JS code for this in `recent_view_ui` and tests.

Reply:
okay @amanagr i thought to removing this button will solve this issue will update the JS and tests